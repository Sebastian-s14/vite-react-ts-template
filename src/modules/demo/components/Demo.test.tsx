import { render } from '@testing-library/react'
import { Demo } from './Demo'

describe('Pruebas en Demo', () => {
  test('Debe de retornar un elemento', () => {
    const { container } = render(<Demo />)
    expect(container).toMatchSnapshot()
  })
  test('Debe retonar el texto DemoApp', () => {
    const { getByText } = render(<Demo />)
    expect(getByText('Demo Component')).toBeTruthy()
  })
})
