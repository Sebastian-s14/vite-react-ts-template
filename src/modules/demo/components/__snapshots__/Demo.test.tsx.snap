// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en Demo Debe de retornar un elemento 1`] = `
<div>
  <div>
    Demo Component
  </div>
</div>
`;
